ci/docker/Dockerfile

```diff
@@ -2,6 +2,14 @@ FROM ubuntu:bionic
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN apt install curl lsb-release && \
+    curl https://dist.apache.org/repos/dist/dev/arrow/KEYS | apt-key add -
+
+RUN tee /etc/apt/sources.list.d/apache-arrow.list <<APT_LINE
+deb [arch=amd64] https://dl.bintray.com/apache/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/ $(lsb_release --codename --short) main
+deb-src https://dl.bintray.com/apache/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/ $(lsb_release --codename --short) main
+APT_LINE
+
 RUN apt update && \
-    apt install -y --no-install-suggests ceph-mon ceph-osd && \
+    apt install -y --no-install-suggests ceph-mon ceph-osd apt-transport-https gnupg libarrow-dev libparquet-dev && \
```

Comment:
hi @ashayshirwadkar , in addition to adding these dependencies here, we also need to uncomment the commented lines in the `ci/workflows/smoke.workflow` file so that these changes can be picked up

Reply:
@ivotron I see following lines commented
```
# NOTE: done offline to speedup travis build time
#
# action "build ceph image" {
#   needs = "download test data"
#   uses = "actions/docker/cli@master"
#   args = "build -t popperized/ceph:luminous ci/docker"
# }
#
# action "registry login" {
#   needs = "build ceph image"
#   uses = "actions/docker/login@master"
#   secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD"]
# }
#
# action "push image" {
#   needs = "registry login"
#   uses = "actions/docker/cli@master"
#   args = "push popperized/ceph:luminous"
# }

action "run tests" {
#  needs = "push image"
  needs = "download test data"
  uses = "docker://popperized/ceph:luminous"
  runs = [
    "sh", "-c", "ci/scripts/run-skyhook-test.sh"
  ]
}
```
Should I uncomment all the these lines?

Reply:
yes, and on the `run tests` action, uncomment `uses = "push image"` and remove `needs = "download test data"`